docs/quickstarts/python-quickstart.md

```diff
@@ -0,0 +1,203 @@
+  // Sends a greeting
+  rpc SayHello (HelloRequest) returns (HelloReply) {}
+}
+
+// The request message containing the user's name.
+message HelloRequest {
+  string name = 1;
+}
+
+// The response message containing the greetings
+message HelloReply {
+  string message = 1;
+}
+```
+Let's update this so that the `Greeter` service has two methods. In your IDE or text editor [what should we recommend, Python people?], open `examples/proto/helloworld.proto` and update it with a new `SayHelloAgain` method, with the same request and response types:
+
+```
+// The greeting service definition.
+service Greeter {
+  // Sends a greeting
+  rpc SayHello (HelloRequest) returns (HelloReply) {}
+  // Sends another greeting
+  rpc SayHelloAgain (HelloRequest) returns (HelloReply) {}
+}
+
+// The request message containing the user's name.
+message HelloRequest {
+  string name = 1;
+}
+
+// The response message containing the greetings
+message HelloReply {
+  string message = 1;
+}
+```
+
+Don't forget to save the file!
+
+## Generate gRPC code
+
+Next we need to update the gRPC code used by our application to use the new service definition. From the `examples/python/helloworld` directory:
+
+```
+$ ./run_codegen.sh
+```
+
+This script invokes the protocol buffer compiler as:
+
+```
+$ protoc -I ../../protos --python_out=. --grpc_out=. --plugin=protoc-gen-grpc=&#96;which grpc&lowbar;python&lowbar;plugin&#96; ../../protos/helloworld.proto</pre>
+```
+
+This generates `helloworld_pb2.py`, which contains our generated client and server classes, as well as classes for populating, serializing, and retrieving our request and response types.
+
+## Update and run the application
+
+We now have new generated server and client code, but we still need to implement and call the new method in the human-written parts of our example application.
+
+### Update the server
+
+In the same directory, open `greeter_server.py`. Implement the new method like this:
+
+```
+class Greeter(helloworld_pb2.BetaGreeterServicer):
+
+  def SayHello(self, request, context):
+    return helloworld_pb2.HelloReply(message='Hello, %s!' % request.name)
+
+  def SayHelloAgain(self, request, context):
+    return helloworld_pb2.HelloReply(message='Hello again, %s!' % request.name)
+
+...
+
+```
+
+### Update the client
+
+In the same directory, open `greeter_client.py`. Call the new method like this:
+
+```
+def run():
+  channel = implementations.insecure_channel('localhost', 50051)
+  stub = helloworld_pb2.beta_create_Greeter_stub(channel)
+  response = stub.SayHello(helloworld_pb2.HelloRequest(name='you'), _TIMEOUT_SECONDS)
+  print("Greeter client received: " + response.message)
+  response = stub.SayHelloAgain(helloworld_pb2.HelloRequest(name='you'), _TIMEOUT_SECONDS)
+  print("Greeter client received: " + response.message)
+```
+
+### Run!
+
+Just like we did before, from the `examples/python/helloworld` directory:
+
+1. Run the server
+
+   ```sh
+   $ python2.7 greeter_server.py &
```

Comment:
Just a note to drop the ampersand here and make this section consistent with the changes discussed above.